src/NavLink.js

```diff
@@ -1,7 +1,14 @@
 import React from 'react'
 import {withRouter} from 'next/router'
+import {Box} from '@primer/components'
 import NodeLink from './NodeLink'
 
-export default withRouter(({is: Tag = NodeLink, href, router, ...rest}) => (
-  <Tag href={href} color="white" px={4} fontWeight={router.pathname === href ? 'bold' : null} {...rest} />
-))
+const NavLink = withRouter(({href, router, ...rest}) => {
```

Comment:
These changes look confusing because this library had a NavLink already, and I replaced it with the NavLink that was in `Section`